project_path: ~/.kyozi
image_topic: /kinect_head/rgb/image_color
joint_states_topic: /joint_states
control_joint_names: [
    r_shoulder_pan_joint, 
    r_shoulder_lift_joint,
    r_upper_arm_roll_joint,
    r_elbow_flex_joint,
    r_forearm_roll_joint,
    r_wrist_flex_joint, 
    r_wrist_roll_joint]
init_joint_names: [
    torso_lift_joint,
    r_shoulder_pan_joint, 
    r_shoulder_lift_joint,
    r_upper_arm_roll_joint,
    r_elbow_flex_joint,
    r_forearm_roll_joint,
    r_wrist_flex_joint, 
    r_wrist_roll_joint,
    l_shoulder_pan_joint, 
    l_shoulder_lift_joint,
    l_upper_arm_roll_joint,
    l_elbow_flex_joint,
    l_forearm_roll_joint,
    l_wrist_flex_joint, 
    l_wrist_roll_joint]
init_joint_angles: [
    0.3,
    -1.3089969389957472,
    0.8726646259971648,
    -1.9198621771937625,
    -1.9198621771937625,
    0.3490658503988659,
    -0.17453292519943295,
    -0.17453292519943295,
    1.3089969389957472,
    0.8726646259971648,
    1.9198621771937625,
    -1.9198621771937625,
    -0.3490658503988659,
    -0.17453292519943295,
    -0.17453292519943295]
